### PR TITLE
Add Wikipedia, Wikidata and network GUID to MVV and VVS

### PR DESCRIPTION
**Pull request by FloEdelmann**

### CODE REVIEW

File: data/transit/route/bus.json [hunk 1]
```diff
@@ -6234,7 +6234,12 @@
       "locationSet": {"include": ["001"]},
       "tags": {
         "network": "Münchner Verkehrs- und Tarifverbund",
+        "network:guid": "DE-BY-MVV",
+        "network:wikidata": "Q259000",
+        "network:wikipedia": "en:Münchner_Verkehrs-_und_Tarifverbund",
```
Comment (FloEdelmann) [inline data/transit/route/bus.json:6239]: Should this rather link to the local (German) Wikipedia page?

File: data/transit/route/light_rail.json [hunk 2]
```diff
@@ -148,7 +148,12 @@
       "locationSet": {"include": ["001"]},
       "tags": {
         "network": "Verkehrs- und Tarifverbund Stuttgart",
+        "network:guid": "DE-BW-VVS",
+        "network:wikidata": "Q2516108",
+        "network:wikipedia": "de:Verkehrs- und Tarifverbund Stuttgart",
```
Comment (FloEdelmann) [inline data/transit/route/light_rail.json:153]: Or should this rather link to the English Wikipedia page? (I copied these three tags from `bus.json`.)

File: data/transit/route/bus.json (hunk 1 above)
Comment (bhousel) [reply data/transit/route/bus.json:6239]: Yes, German stuff should link to the German wikipedia page, if there is one.

A few other things I noticed:
- make sure to update the `locationSet` to only include "de"
- replace the underscores with spaces in the wikipedia sitelinks

**Review by bhousel — CHANGES_REQUESTED** [review]
Welcome @FloEdelmann 👋 
Looks like you've really done your research, thank you! 
Just a few things to change:
* make sure to update the locationSet to only include "de"
* replace the underscores with spaces in the wikipedia sitelinks

**Comment by Karthoo** [comment]
I would also recommend to add "network:short": "MVV" (/VVS) and remove the operator tags because the lines within the networks are operated by sub-contractors

**Comment by FloEdelmann** [comment]
> I would also recommend to add "network:short": "MVV" (/VVS)

@Karthoo I considered this, as I've seen it for other networks, but the tag is not documented in the OSM wiki. I'll add it now, though.

> and remove the operator tags because the lines within the networks are operated by sub-contractors

Alright, will do.

**Review by bhousel — APPROVED** [review]

**Comment by bhousel** [comment]
Thanks for the quick fixes @FloEdelmann 👍

**Pull request merged**